cedar-policy/CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- `ValidatorActionId::applies_to_principals` and `ValidatorActionId::applies_to_resources`
```

Comment:
We haven't been tracking changes to internal APIs here

Reply:
ack